File: exercism/javascript/bob/bob.js
/*jslint node: true */
"use strict";

var _ = require('lodash');

module.exports = Bob;

function Bob() {
  this.hey = function (phrase) {
    var string = phrase.split('');
    if ((string[phrase.length-1]) === '?') {
      return "Sure.";
    }
    else{
      return "Whatever.";
    }
  };
}

// Bob is a lackadaisical teenager. In conversation, his responses are very limited.
//
// Bob answers 'Sure.' if you ask him a question.
//
// He answers 'Woah, chill out!' if you yell at him.
//
// He says 'Fine. Be that way!' if you address him without actually saying anything.
//
// He answers 'Whatever.' to anything else.

File: exercism/python/bob/bob.py
def hey(phrase):
  lastchar = phrase[-1:]
  if (lastchar == '!'):
    return 'Woah, chill out!'
  elif (len(phrase) == 0):
    return 'Fine. Be that way!'
  else:
    return 'whatever'

def yell(phrase):
  return 'Woah, chill out!'



if __name__ == "__main__":
  print "hello"
  print hey("")
  print hey("hey!")
